src/module/actor/sheets/SR5BaseActorSheet.ts

```diff
@@ -860,11 +932,20 @@ export class SR5BaseActorSheet extends foundry.appv1.sheets.ActorSheet {
      * Reset all condition tracks to zero values.
      * @param event
      */
-    async _onClearConditionTrack(event) {
+    static async #clearConditionMonitor(this: SR5BaseActorSheet, event) {
         event.preventDefault();
 
-        const track = $(event.currentTarget).closest('.horizontal-cell-input').data().id;
-        const data = {};
+        const track = event.target.closest('[data-id]').dataset.id;
+
+        const uuid = event.target.closest('[data-uuid]')?.dataset?.uuid ?? '';
+        if (track === 'matrix' && uuid) {
+            const document = await fromUuid(uuid);
+            if (document && document instanceof SR5Item) {
+                return await document.setMatrixDamage(0);
```

Comment:
Funnily using `return await` has less performance than just using `return`, because it don't have to wait twice. But I don't know how it works with handlers, does it wait the function to finish? I always assumed it's a synced function.